openwrt-mdnsd-cve-2020-11750/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenWRT mdns CVE-2020-11750
+
+Three out-of-bounds access issues were found in OpenWRT's mdns. They were reported to the OpenWRT security address on April 9 2020, and a fix ([1](https://git.openwrt.org/?p=project/mdnsd.git;a=commit;h=e74a3f9883199e9db7220d52b78e5fbdb4441ca3), [2](https://git.openwrt.org/?p=project/mdnsd.git;a=commit;h=cdac0460ba50dc45735f0be2e19a5a8efc3dafe1)) was released soon after.
+
+The code included in this project is an edited version of the original OpenWRT mdns code and `__dn_expand` from musl libc's DNS resolution code.
+
+## To build
+
+Assuming you just want to build the docker image, run:
+
+```bash
+docker build -t forallsecure/openwrt-mdnsd-cve-2020-11750 .
+```
+
+## Get from Dockerhub
+
+If you don't want to build locally, you can pull a pre-built image
+directly from dockerhub:
+
+```bash
+docker pull forallsecure/openwrt-mdnsd-cve-2020-11750
+```
+
+
+## Run under Mayhem
+
+Change to the `openwrt-mdnsd-cve-2020-11750` folder and run:
+
+```bash
+mayhem run mayhem/mdnsd
+```
+
+and watch Mayhem replicate the bugs! These bugs take seconds!
+
+## Run locally
+
+You can run this locally by running the libfuzzer harness or standalone harness
+with qemu-afl inside the docker container.
+
+## POC
+
+We have included a proof of concept output under the `poc`
+directory.
+
+> Note: Fuzzing has some degree of non-determinism, so when you run
```

Comment:
Copy-paste bug. Please remove.

Reply:
Remove this whole section or just the reference to Oniguruma regex?